Compilation of Skeleton fails due to deprecated XiaData calls

I tried to compile utkscan linking with hhirf libraries and it is not compiling

```
[ 28%] Building CXX object Scan/util/source/CMakeFiles/skeleton.dir/Skeleton.cpp.o
/home/pixie16/DPL/paass/Scan/util/source/Skeleton.cpp: In member function ‘virtual void skeletonUnpacker::ProcessRawEvent(ScanInterface*)’:
/home/pixie16/DPL/paass/Scan/util/source/Skeleton.cpp:42:35: error: ‘class XiaData’ has no member named ‘modNum’
   count1cc_(8000, (current_event->modNum*16+current_event->chanNum), 1);
                                   ^
/home/pixie16/DPL/paass/Scan/util/source/Skeleton.cpp:42:60: error: ‘class XiaData’ has no member named ‘chanNum’
   count1cc_(8000, (current_event->modNum*16+current_event->chanNum), 1);
                                                            ^
make[2]: *** [Scan/util/source/CMakeFiles/skeleton.dir/Skeleton.cpp.o] Error 1
```

--- a/Scan/util/source/Skeleton.cpp
+++ b/Scan/util/source/Skeleton.cpp
@@ -39,7 +39,7 @@ void skeletonUnpacker::ProcessRawEvent(ScanInterface *addr_/*=NULL*/){
 
 #ifdef USE_HRIBF		
 		// If using scanor, output to the generic histogram so we know that something is happening.
-		count1cc_(8000, (current_event->modNum*16+current_event->chanNum), 1);
+		count1cc_(8000, (current_event->GetId()), 1);
 #endif	
 	
 		// Check that this channel event exists.

